solidity/deploy/00_resolve_tbtc_v1_token.ts

```diff
@@ -0,0 +1,28 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments, helpers } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCToken = await deployments.getOrNull("TBTCToken")
```

Comment:
Don't we want to use `get()` here as well?

Reply:
Nope, `get` will throw in case the `TBTCToken` deployment doesn't exist. We want to handle this case by deploying a stub.